### PR TITLE
Pipe does not work in windows makefile with Mingw and Msys so we came up with another solution that does.  This also removes the last remaining dependency on msys (sed) as far as we can tell.

### PR DESCRIPTION
**Pull request by powerpbx**

### CODE REVIEW

**Comment by powerpbx** [comment]
This is just our first try.  If anyone has a better way to do this we we are open to suggestions.

File: src/net/java/sip/communicator/impl/version/NightlyBuildID.java [hunk 1]
```diff
@@ -27,5 +27,5 @@
     /**
      * The build ID.
      */
-    public static final String BUILD_ID="0.build.by.SVN";
+    public static final String BUILD_ID="1";
 }
```
Comment (damencho) [inline src/net/java/sip/communicator/impl/version/NightlyBuildID.java]: When invoking ant you just need to pass -Dlabel="1" and this is going to be set as build version.

**Comment by ibauersachs** [comment]
You should work on a branch, not on master. And for a pull request you *need* to do that.

File: resources/install/build.xml [hunk 2]
```diff
@@ -573,7 +573,7 @@
                 todir="${light.dir}"
                 overwrite="true" />
 
-        <property name="wix.home" value="C:\Program Files\WiX" />
+        <property name="wix.home" value="C:\wix310" />
```
Comment (damencho) [inline resources/install/build.xml]: This also can be controlled with passing parameters to ant as -Dwix.home="C:\wix310".

**Comment by powerpbx** [comment]
Thanks for the info.

**Pull request closed without merging**